### DISCUSSION TITLE
Search engine - creating an order

### DISCUSSION THREAD

**Discussion by anita-coody** [PrestaShop/PrestaShop#35436] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

In the shopping cart in the backend of the store, the search engine works unintuitively. 

For example, a product with the name "test 12345" does not scrub when you type the phrase "test". The product appears when you type "test 1". We make no secret of the fact that this is problematic, and customers have already commented to us more than once on the way this search engine works.

### Expected behavior

We expect the search engine in the shopping cart backend to function intuitively, allowing for efficient and accurate product retrieval based on relevant keywords. 

### Steps to reproduce

1. go to the "Orders" tab
2. click "Add new order"
3. search for a customer
4. go to the product search engine 

### PrestaShop version(s) where the bug happened

8.1.0

### PHP version(s) where the bug happened

7.4.30

**Comment by paulnoelcholot** [comment]
Hello @anita-coody,

Thank you for your suggestion. The Maintainer Team will take it into consideration for future developments. 

Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you create it. 

If you or anyone else is willing to [contribute this feature](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/) to the PrestaShop Project, the teams will be happy to help by ensuring that it fits well within the project. Please let us know if that is the case!

Thank you
